app/Domain/Integrations/FormRequests/UpdateContactInfoRequest.php

```diff
@@ -27,9 +27,9 @@ public function rules(): array
                 'technical.type' => ['string', 'min:2', 'max:255'],
                 'technical.firstName' => ['string', 'min:2', 'max:255'],
                 'technical.lastName' => ['string', 'min:2', 'max:255'],
-                'firstname' => ['nullable', 'string', 'min:2', 'max:255'],
-                'lastname' => ['nullable', 'string', 'min:2', 'max:255'],
-                'email' => ['nullable', 'string', 'email', 'min:2', 'max:255'],
+                'newContributorFirstName' => ['nullable', 'string', 'min:2', 'max:255'],
```

Comment:
For me it feels a bit weird to read `new` inside an `update` statement.

Reply:
I agree the flow is a bit strange. I'll take a look at the new urls and new contributor flow in a whole. Because it can be improved.